aptos-move/aptos-aggregator/src/aggregator_extension.rs

```diff
@@ -393,39 +409,118 @@ mod test {
         let mut aggregator_data = AggregatorData::default();
         aggregator_data.create_new_aggregator(aggregator_id_for_test(200), 200);
 
-        let aggregator = aggregator_data.get_aggregator(aggregator_id_for_test(200), 200);
+        let aggregator =
+            aggregator_data.get_aggregator(aggregator_id_for_test(200), 200, &*TEST_RESOLVER, true);
         assert_ok!(aggregator.add(100));
         assert_ok!(aggregator.read_and_materialize(&*TEST_RESOLVER, &aggregator_id_for_test(200)));
         assert_eq!(aggregator.value, 100);
     }
 
+    #[test]
+    fn test_materialize_known_aggregator_disabled() {
+        let mut aggregator_data = AggregatorData::default();
+        aggregator_data.create_new_aggregator(aggregator_id_for_test(200), 200);
+        let aggregator =
+            aggregator_data.get_aggregator(aggregator_id_for_test(200), 200, &*TEST_RESOLVER, true);
```

Comment:
This should be
```
aggregator_data.get_aggregator(aggregator_id_for_test(200), 200, &*TEST_RESOLVER, false);
```
with aggregator disabled?

Reply:
In the test, I am doing `get_aggregator` twice. I did some operations with the flag set to `true` first. When the flag is set to `false` latter, I was checking out if the aggregator value is initialized with the correct value. Am I doing wrong?

Reply:
All good! Too many tests haha (which is great)